src/Cluster/Health.php

```diff
@@ -60,6 +84,7 @@ public function getClusterName(): string
      * Gets the status of the cluster.
      *
      * @return string green, yellow or red
+     * @phpstan-return HealthStatus
```

Comment:
We could use `@return 'green'|'yellow'|'red'` instead of introducing virtual class ;-)

see: https://phpstan.org/writing-php-code/phpdoc-types#literals-and-constants